src/ConfigProvider.php

```diff
@@ -9,7 +11,7 @@ class ConfigProvider
      *
      * @return array
      */
-    public function __invoke()
+    public function __invoke(): array
```

Comment:
This class is not `final`, so we can't add return type declarations, as they may break subtypes